src/main/kotlin/features/inventory/PriceData.kt

```diff
@@ -60,7 +60,7 @@ object PriceData : FirmamentFeature {
 
 	@Subscribe
 	fun onItemTooltip(it: ItemTooltipEvent) {
-		if (!TConfig.tooltipEnabled && !TConfig.enableKeybinding.isPressed()) {
+		if ((!TConfig.tooltipEnabled && !TConfig.enableKeybinding.isBound) || (TConfig.tooltipEnabled && TConfig.enableKeybinding.isBound && !TConfig.enableKeybinding.isPressed())) {
```

Comment:
Can you split this up into multiple guard statements?
```suggestion
		if (!TConfig.tooltipEnabled) return
		if (TConfig.enableKeybinding.isBound && !TConfig.enableKeybinding.isPressed())
```

Reply:
Also just for the record. the old behaviour is intended.